CHANGELOG.latest.md

```diff
@@ -1,31 +1,29 @@
-- Second RC for RevenueCat framework v4 🎉
```

Comment:
This is the only file really worth reviewing, the rest is all automatically-generated

Reply:
we'll improve this soon with static DocC site generation + moving to a bucket or a different place, this is a temporary annoyance